stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1674,9 +1673,8 @@ internal constructor(
             "[queryChannelsInternal] userId: $userId, scopedUserId: $scopedUserId, " +
                 "isConnectionRequired: $isConnectionRequired, request: $request"
         }
-        return callPostponeHelper.postponeCallIfNeeded(shouldPostpone = isConnectionRequired) {
-            api.queryChannels(request)
-        }
+
+        return api.postponeCallIfNeeded(shouldPostpone = isConnectionRequired) { api.queryChannels(request) }
```

Comment:
As far as `isConnectionRequired` is included on the `request` object, why don't we handle it within `api.queryChannels()`?
Then we could remove the method `postponeCallIfNeeded()`